pkg/scheduler/core/generic_scheduler.go

```diff
@@ -429,7 +427,7 @@ func (g *genericScheduler) prioritizeNodes(
 
 	if klog.V(10).Enabled() {
 		for i := range result {
-			klog.Infof("Host %s => Score %d", result[i].Name, result[i].Score)
+			klog.InfoS("Host => Score", "Host", result[i].Name, "Score", result[i].Score)
```

Comment:
Active voice: who is doing what?

camel case

Reply:
I will use "The node's final score" instead.

Reply:
The message needs to have an actor and an action

Reply:
klog.InfoS("combined (added) all scores together for that node", "node", result[i].Name, "score", result[i].Score)

Reply:
How about "klog.InfoS("Combined (added) all scores together for that node", "node", result[i].Name, "score", result[i].Score)"? Or do you have better advice?